war/src/main/webapp/scripts/hudson-behavior.js

```diff
@@ -2222,7 +2220,7 @@ function updateBuildHistory(ajaxUrl,nBuild) {
 
                     // delete all rows
                     var searchRow;
-                    if (Element.hasClassName(rows[0], "build-search-row")) {
+                    if (rows[0].classList.contains('build-search-row')) {
```

Comment:
Is it certain that `rows[0]` is not undefined?

Reply:
It must be given it was also used before?